dev/com.ibm.ws.jpa_testframework/src/com/ibm/ws/testtooling/vehicle/web/AbstractDatabaseManagementServlet.java

```diff
@@ -162,115 +163,114 @@ private static void executeDDL(DataSource ds, UserTransaction tx, ClassLoader cl
 
         final PrintWriter pw = resp.getWriter();
         try {
+            // Set transaction timeout to 10 minutes
+            tx.setTransactionTimeout((int) Duration.ofMinutes(10).getSeconds());
             tx.begin();
-            final Connection conn = ds.getConnection();
-            final DatabaseMetaData dbMeta = conn.getMetaData();
-            dumpDBMeta(dbMeta);
 
-            final Statement stmt = conn.createStatement();
-            final String defaultSchemaName = (overrideDefaultSchema == null || "".equals(overrideDefaultSchema.trim())) ? //
-                            sanitize(dbMeta.getUserName()) : // Usually the default schema name
-                            sanitize(overrideDefaultSchema); // But always allow for test client to override
+            try (Connection conn = ds.getConnection()) {
+                final DatabaseMetaData dbMeta = conn.getMetaData();
+                dumpDBMeta(dbMeta);
 
-            int totalCount = 0, successCount = 0;
-            int cmdIdx = 0;
-            for (String command : commands) {
+                final String defaultSchemaName = (overrideDefaultSchema == null || "".equals(overrideDefaultSchema.trim())) ?
+                        sanitize(dbMeta.getUserName()) :
+                        sanitize(overrideDefaultSchema);
 
-                // Check if more than 1 minute has passed
-                if ((System.currentTimeMillis() - timestart) > 60000) {
-                    System.out.println("More than 1 minute has passed. Committing and beginning a new transaction.");
+                int totalCount = 0, successCount = 0;
+                int cmdIdx = 0;
 
-                    // Commit the current transaction and begin a new one
-                    tx.commit();
-                    tx.begin();
+                for (String command : commands) {
+                    final String sql = command.replace("${schemaname}", defaultSchemaName).trim();
+                    if ("".equals(sql)) {
+                        continue;
+                    }
+                    if (sql.startsWith("#")) {
+                        // Commented out sql line
+                        continue;
+                    }
```

Comment:
```suggestion
                    if ("".equals(sql) || sql.startsWith("#")) {
                        continue;
                    }
```

suggestion: easier readability